lib/takeaway.rb

```diff
@@ -0,0 +1,33 @@
+
+class Takeaway 
```

Comment:
Currently this class is fulfilling more than one responsibility:
- storing and displaying the menu
- keeping track of an order

Can you see a way we can refactor so that we have multiple classes which each follow SRP?